runtime/service_integ_test.go

```diff
@@ -1104,3 +1104,116 @@ func TestUpdateVMMetadata_Isolated(t *testing.T) {
 	t.Logf("stdout output from task %q: %s", containerName, stdout)
 	assert.Equalf(t, "45", stdout, "container %q did not emit expected stdout", containerName)
 }
+
+// TestRandomness validates that there is a reasonable amount of entropy available to the VM and thus
+// randomness available to containers (test reads about 2.5MB from /dev/random w/ an overall test
```

Comment:
That's a lot of data to pull from /dev/random. How did you arrive at that number?

Reply:
I wanted to provide a fair number of blocks to `rngtest`, so I started at this arbitrary number (1024 blocks). I also wanted to read enough so that we can easily catch the container getting blocked on reading `/dev/random` when the entropy isn't being seeded by `haveged`.

When I ran this test w/out `haveged`, I did indeed see the container get blocked and thus the test timeout+fail, so that worked as expected. When `haveged` is present, the test runs relatively quickly so I didn't see any reason to change the arbitrary number I started at. Is there any reason to that you see?

Reply:
Nope. It still feels like a lot to me. Applications would typically not read more than a few kB at most. But if the test passes with 2.5 MB, then let's go with it.